main.go

```diff
@@ -1,11 +1,25 @@
 package main
 
 import (
+	"github.com/gin-gonic/gin"
+	"learningbay24.de/backend/api"
 	"learningbay24.de/backend/config"
 )
 
 func main() {
 	config.InitConfig()
 	// TODO: do something with the db handle
 	_ = config.SetupDbHandle()
-}
+	router := gin.Default()
+	router.GET("/courses/:id",api.GetCourseById)
+	router.GET("user/:user_id/courses",api.GetUserCourses)
+	router.GET("courses/:id/users",api.GetUsersInCourse)
+	router.DELETE("/courses/:id/delete",api.DeleteCourseById)
```

Comment:
Nach REST sollte `DELETE` auf dem gleichen Pfad wie `GET` passieren, also beides auf `/courses/:id`. Die `GET` bzw. `DELETE` HTTP Methoden sagen schon, was damit passiert, das muss nicht extra in der URL stehen. Bitte abändern.